test/e2e-common.sh

```diff
@@ -21,9 +21,10 @@ source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/e2e-tests.sh
 # If gcloud is not available make it a no-op, not an error.
 which gcloud &> /dev/null || gcloud() { echo "[ignore-gcloud $*]" 1>&2; }
 
-# Let sed work on mac.
+# Use GNU tools on macOS. Requires the 'grep' and 'gnu-sed' Homebrew formulae.
```

Comment:
Call for opinions: should we fail early with an error message when those commands are not available in the developer's environment?